dags/common/parsing/json_extractors.py

```diff
@@ -0,0 +1,30 @@
+from aps.utils import get_nested
+from common.parsing.extractor import IExtractor
+
+
+class ValueExtractor(IExtractor):
```

Comment:
I find this class' name a bit confusing. Why do we have multiple keys ? Are we setting a value from multiple keys ? I understood that we are getting data from nested elements, but I had to dive deep in the code to understand. IMO we should change this class to extract only the value (IE one destination and one source) and add a class named "NestedValueExtractor". Instead of `keys` I would name this `json_path`. What do you think ?